package.json

```diff
@@ -17,7 +17,7 @@
     "dev:vscode": "lerna run dev --scope astro-languageserver --scope astro-vscode --scope @astrojs/parser --parallel --stream",
     "format": "prettier -w \"**/*.{js,jsx,ts,tsx,md,json}\"",
     "lint": "eslint \"packages/**/*.ts\"",
-    "test": "lerna run test --scope astro --scope prettier-plugin-astro --stream",
+    "test": "lerna run test --scope astro --stream",
```

Comment:
```suggestion
    "test": "yarn workspace astro run test",
```

Reply:
Is there a way to specify workspaces like lerna with `--scope astro --scope prettier-plugin-astro`? Would you write that `yarn workspace astro prettier-plugin-astro run test`?